README.md

```diff
@@ -1,5 +1,9 @@
 # SPV WALLET WEB FRONTEND
 
+## Documentation
```

Comment:
remove `Documentation` header. move the sentence at the end of paragraph

Reply:
Done